src/components/scenes/ManageTokensScene.js

```diff
@@ -74,11 +75,15 @@ export default class ManageTokens extends Component<ManageTokensProps, State> {
 
   render () {
     const { metaTokens, name } = this.props.guiWallet
-    const { manageTokensPending } = this.props
+    const { manageTokensPending, plugins } = this.props
     const accountMetaTokenInfo = [...this.props.settingsCustomTokens]
     const combinedTokenInfo = UTILS.mergeTokensRemoveInvisible(metaTokens, accountMetaTokenInfo)
-
-    const sortedTokenInfo = combinedTokenInfo.sort((a, b) => {
+    const allCurrencyInfos = plugins.allCurrencyInfos
+    const allSupportedParentCurrencies = allCurrencyInfos.map(info => info.currencyCode)
+    const combinedFilteredTokenInfo = combinedTokenInfo.filter(token => {
+      return !allSupportedParentCurrencies.includes(token.currencyCode)
+    })
+    const sortedTokenInfo = combinedFilteredTokenInfo.sort((a, b) => {
```

Comment:
This isn't needed, since we prevent tokens with conflicting currency codes from even entering the system, so we don't need to filter them out.